html/modules/custom/rwr_sitrep/rwr_sitrep.post_update.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Post update hooks hooks.
+ */
+
+/**
+ * Add default taxonomy term for cluster_or_working_group_type.
+ */
+function rwr_sitrep_post_update_add_terms() {
+  // Add default terms.
+  $term = rwr_sitrep_create_term_if_needed('cluster_or_working_group_type', 'Situation Report');
+  $term->set('field_pdf_enabled', TRUE);
+  $term->save();
+
+  $term = rwr_sitrep_create_term_if_needed('cluster_or_working_group_type', 'Cluster');
```

Comment:
@attiks The site admins were thinking of using this taxonomy for Cluster types, I think, rather than to separate Clusters, Working Groups and Sectors. We could create these terms, but it might be confusing.

Reply:
I missed that somehow, feel free to adapt/remove it